src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -506,11 +484,11 @@ private void determineTerritoriesThatCanBeHeld(final List<ProTerritory> prioriti
 
         // Determine counter attack results to see if I can hold it
         final ProBattleResult result2 =
-            battleUtils.calculateBattleResults(player, t, patd.getMaxEnemyUnits(), remainingUnitsToDefendWith,
+            ProBattleUtils.calculateBattleResults(player, t, patd.getMaxEnemyUnits(), remainingUnitsToDefendWith,
                 enemyAttackOptions.getMax(t).getMaxBombardUnits(), false);
         final boolean canHold =
             (!result2.isHasLandUnitRemaining() && !t.isWater()) || (result2.getTUVSwing() < 0)
-                || (result2.getWinPercentage() < MIN_WIN_PERCENTAGE);
+                || (result2.getWinPercentage() < ProData.minWinPercentage);
```

Comment:
Same here, see if ProData could compute this for you,  "public static boolean canHold( result )"

(This would also make the logic very testable too!)